WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -563,6 +564,8 @@
         expect(post.body).to include "World records: Jeremy Fleischman 3x3 one-handed 50.00 (average), Vincent Sheu (2006SHEU01) 3x3 fewest moves 25 (single), 3x3 fewest moves 26.00 (average), Vincent Sheu (2006SHEU02) 2x2 Cube 10.00 (single)"
         expect(post.body).to include "North American records: Jeremy Fleischman 3x3 one-handed 41.00 (single), 3x3 one-handed 40.00 (single)"
         expect(post.title).to include "in #{competition.cityName}, #{competition.countryId}"
+        competition.reload
+        expect(competition.results_posted).to be_within(1.second).of before_posting
```

Comment:
The right way to test time dependent code is by mocking out the system clock.

I've used Timecop, and I think it's simple enough to add it to the project for this test: https://github.com/travisjeffery/timecop